.travis.yml

```diff
@@ -3,6 +3,9 @@ language: python
 # Use the new container-based Travis infrastructure.
 sudo: false
 
+# Make sure we are on Ubuntu 14.04
+dist: trusty
```

Comment:
Please explain in the commit-message why we require this. I assume it's since tests fail on 12.04. But this change also prohibits using newer versions. Thus we should state reason.